src/lib/table/table.spec.ts

```diff
@@ -7,17 +7,17 @@ import {MdTableModule} from './index';
 import {MdTable} from './table';
 
 describe('MdTable', () => {
-  let fixture: ComponentFixture<SimpleMdTableApp>;
+  let fixture: ComponentFixture<MdTableApp>;
```

Comment:
* Add test case for `when` missing on multiple rows?
* Add test case for using the default row def?

There's also no test now for a table that just has one row def

Reply:
Added test for missing when on multiple rows. Default row def is used for any case where there is no `when`, e.g. `SimpleCdkTableApp`

Note, majority of testing for the when logic is in `cdk/table.ts`. Goal of the `lib/table.ts` test is to show that the `MdRowDef` successfully reads the `when` input and passes it on to the `CdkRowDef`.